functional_test/src/test/java/test/newrelic/test/agent/spans/SpanErrorsTest.java

```diff
@@ -98,6 +101,7 @@ public void testNoticedStringDoesNotBubble() {
         ));
     }
 
+    @Category(IBMJ9IncompatibleTest.class)
     @Test
     public void testNoticedErrorOverridesThrownError() {
         runTransaction(new SpanErrorFlow.NoticeAndThrow());
```

Comment:
IBMj9 throws this as `divide by zero` and our tests look for the String `/ by zero`.